reflex/constants/base.py

```diff
@@ -120,6 +120,8 @@ class Next(SimpleNamespace):
     PACKAGE_LOCK = "package-lock.json"
     # Regex to check for message displayed when frontend comes up
     FRONTEND_LISTENING_REGEX = "Local:[\\s]+(.*)"
+    # Place holder domain name
+    PLACEHOLDER_DOMAIN = "http://example.com"
```

Comment:
we do not need the placeholder domain if we build the complete sitemap with python